src/applications/personalization/profile/components/notification-settings/NotificationSettings.jsx

```diff
@@ -1,20 +1,122 @@
-import React, { useEffect } from 'react';
+import React from 'react';
```

Comment:
The changes in this file, as well as the new supporting components, are the real meat of this PR